index.php

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Assignment 7</title>
+
+     <!--Index.php
+    Assignment 7
+    INFO 2340 
+    Donjeta Emerllahu-->
+
+</head>
+<body>
+    <?php //define variables here.
+    $my_dinner= 'Pizza';
+    $_num = 2;
+    $fav_place='Joslyn art museum';
+    $fav_animal='lion';
+
+    ?>
+    <h1>Good morning, Nebraska</h1>
+    <div>Last night I ate <?php Echo "$my_dinner";?>.</div>
```

Comment:
You wouldn't need to have the variable in quotes here. Just echo $my_dinner; should output fine.